WalletWasabi.Documentation/WasabiSetupRegtest.md

```diff
@@ -94,18 +96,19 @@ Todo:
 1. Go to `WalletWasabi\WalletWasabi.Fluent.Desktop` folder.
 2. Open the command line and run the Wasabi Client with:
 `dotnet run --no-build`
-3. Go to Tools/Settings and set the network to RegTest
+3. Go to Settings/Bitcoin and set the network to RegTest
 4. Close Wasabi and restart it with:
 `dotnet run --no-build`
 5. Generate a wallet in Wasabi named: R1.
-6. Generate a receive address in Wasabi, now go to Bitcoin Core gui to the Send tab.
+6. Generate a receive address in Wasabi, now go to Bitcoin Knots to the Send tab.
 7. Send 1 BTC to that address.
 8. Open another Wasabi instance from another command line:
 `dotnet run --no-build`
```

Comment:
How this is possible? Single instance checker shouldn't allow it.

Reply:
That is true. This step should simply get removed.

Reply:
Deleted it.